src/gui/mzroll/mzfileio.cpp

```diff
@@ -223,112 +223,169 @@ PK$PEAK: m/z int. rel.int.
     } while (!line.isNull());
     return compoundCount;
 }
-//TODO: Shouldnot be here
-int mzFileIO::loadNISTLibrary(QString fileName) {
-    qDebug() << "Loading Nist Libary: " << fileName;
-    QFile data(fileName);
 
+int mzFileIO::loadNISTLibrary(QString filepath)
```

Comment:
mzFileIO is not available to the core and therefore not to the CLI. In the future, we would like to add support for MS/MS data in CLI as well. Why was the decision made to add it here and not in one of the database files? I understand we will have to merge database.cpp and databases.cpp to prevent code duplication, but there are no such plans for mzFileIO as of now.